src/test/java/com/philips/research/spdxbuilder/persistence/tree/TreeFormatsTest.java

```diff
@@ -83,6 +83,22 @@ void gradle() {
                     new Package("com.group", "upgraded", "2.3.4"));
         }
 
+        @Test
+        void nuget() {
+            format.configure(parser, "nuget");
+
+            parse("ProjectName 1.0.0-dev",
+                    "---IdentityModel.AspNetCore 3.0.0.0",
```

Comment:
Would be a good idea to add a sample of the "... by ..." format to make sure there are no surprise matches.

Reply:
I agree, I added a package with this odd format.